app/javascript/dashboard/components/widgets/conversation/components/FilterInput.vue

```diff
@@ -1,40 +1,32 @@
 <template>
   <div class="filters">
-    <div class="filter--attributes">
-      <select
-        v-model="attribute_key"
-        class="filter--attributes_select"
-        @change="resetFilter()"
-      >
-        <option
-          v-for="attribute in filterAttributes"
-          :key="attribute.key"
-          :value="attribute.key"
+    <div class="filter">
+      <div class="filter-inputs">
+        <select
+          v-model="attributeKey"
+          class="filter__question"
+          @change="resetFilter()"
         >
-          {{ $t(`FILTER.ATTRIBUTES.${attribute.attributeI18nKey}`) }}
-        </option>
-      </select>
-      <button class="filter--attribute_clearbtn" @click="removeFilter">
-        <i class="icon ion-close-circled" />
-      </button>
-    </div>
-    <div class="filter-values">
-      <div class="row">
-        <div
-          class="columns padding-right-small"
-          :class="showUserInput ? 'small-4' : 'small-12'"
-        >
-          <select v-model="filter_operator" class="filter--values_select">
-            <option
-              v-for="(operator, o) in operators"
-              :key="o"
-              :value="operator.value"
-            >
-              {{ $t(`FILTER.OPERATOR_LABELS.${operator.value}`) }}
-            </option>
-          </select>
-        </div>
-        <div v-if="showUserInput" class="small-8 columns">
+          <option
+            v-for="attribute in filterAttributes"
+            :key="attribute.key"
+            :value="attribute.key"
+          >
+            {{ $t(`FILTER.ATTRIBUTES.${attribute.attributeI18nKey}`) }}
```

Comment:
Small change here to remove this hardcoded value `{{ $t(`FILTER.ATTRIBUTES.${attribute.attributeI18nKey}`) }}` should be replaced with `{{ attribute.name }}` to make it usable. I had raised another PR for this [one](https://github.com/chatwoot/chatwoot/pull/3440/files)

Reply:
Should I change it here too?

Reply:
@fayazara Please resolve this while taking a downmerge.